src/js/popup.js

```diff
@@ -430,9 +442,14 @@ Logic.registerPanel(P_ONBOARDING_2, {
   initialize() {
     // Let's move to the containers list panel.
     [...document.querySelectorAll(".onboarding-next-button")].forEach(nextElement => {
-      Logic.addEnterHandler(nextElement, async () => {
-        await Logic.setOnboardingStage(2);
-        Logic.showPanel(P_ONBOARDING_3);
+      Logic.addEnterHandler(nextElement.parentNode, async (e) => {
+        if(e.target === nextElement
```

Comment:
This code appears to be duplicated with all Enter handlers:
- What does it do?
- Why isn't it part of that method?